stable/ambassador/values.yaml

```diff
@@ -0,0 +1,126 @@
+# Default values for ambassador.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+daemonSet: false
+
+ambassador:
+  id: default
+
+namespace:
+  single: false
+  # name: default
+
+# Additional container environment variable
+env:
+  {}
+  # Exposing statistics via StatsD
```

Comment:
Seems like these comments are redundant since it's already added if `prometheusExporter.enabled=true`

Reply:
Do you think it makes more sense to set them automatically if the prometheus sidecar is enabled, or  should they be manually specified with values?

Reply:
Also the `STATSD_ENABLED` is relevant if using for example datadog. And the `STATSD_HOST` can be used to specify another host than the default `statsd-sink` if not using the sidecar.

Reply:
I think it's fine to set them automatically depending upon the sidecar enabled.